designer/client/src/components/modals/ValidationLabels.tsx

```diff
@@ -43,15 +43,14 @@ export default function ValidationLabels(props: Props) {
             const { errorCode, templateValues, message } = validationError;
 
             const canRenderI18nTemplate = errorCode != null && templateValues != null;
```

Comment:
We can simplify it to `const canRenderI18nTemplate = Boolean(errorCode && templateValues)`